scipy-notebook/Dockerfile

```diff
@@ -74,6 +74,10 @@ RUN conda create --quiet --yes -p $CONDA_DIR/envs/python2 python=2.7 \
 RUN ln -s $CONDA_DIR/envs/python2/bin/pip $CONDA_DIR/bin/pip2 && \
     ln -s $CONDA_DIR/bin/pip $CONDA_DIR/bin/pip3
 
+# Import matplotlib the first time to build the font cache.
+ENV XDG_CACHE_HOME /home/$NB_USER/.cache/
+RUN $CONDA_DIR/envs/python2/bin/python -c "import matplotlib.pyplot"
```

Comment:
Didn't know you could control the backend via env var. Cool.

`python -c` definitely won't work, and that's why I was thinking running `ipython -c` might do the trick instead.